src/index.html

```diff
@@ -19,9 +19,400 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
The 'preconnect' link to 'https://fonts.gstatic.com' is duplicated. You should remove one of these to avoid unnecessary requests.

Reply:
Good point 🔥 Fix it